fundrebalancer/fundrebalancer.go

```diff
@@ -472,6 +480,16 @@ func (r *FundRebalancer) GetRebalanceTxns(
 				return nil, fmt.Errorf("hex decoding evm call data: %w", err)
 			}
 
+			if err := r.ERC20Approval(ctx, SkipGoTxnWithMetadata{
```

Comment:
did this move because you were getting an error that you can't do the estimate gas call because it exceeds the allowance? just curious because im also seeing that with my pr

Reply:
yessir